src/app/state/lastCompositionEnd.ts

```diff
@@ -0,0 +1,3 @@
+import { atom } from 'jotai';
+
+export const lastCompositionEndAtom = atom<number>(Number.NEGATIVE_INFINITY);
```

Comment:
since we do not know the initial value could we set it to `undefined`

Reply:
It is done!